tests/unit/test-class-wc-payment-gateway-wcpay.php

```diff
@@ -3969,6 +3969,50 @@ public function is_proper_intent_used_with_order_returns_false() {
 		$this->assertFalse( $this->card_gateway->is_proper_intent_used_with_order( WC_Helper_Order::create_order(), 'wrong_intent_id' ) );
 	}
 
+
+	public function test_get_recommended_payment_method() {
+		$this->mock_wcpay_account
+			->expects( $this->once() )
+			->method( 'get_recommended_payment_methods' )
+			->with( 'US' );
+		$this->card_gateway->get_recommended_payment_methods( 'US' );
```

Comment:
Shouldn't we assert that we receive what the account `get_recommended_payment_methods` method returns?

Reply:
not really, I wanted to test only `get_recommended_payment_methods`   - what is called inside the method. Anywway I added also assert to the method

Reply:
Isn't the purpose of a unit test to mock the externals and assert the outputs?